src/pages/Home/Home.tsx

```diff
@@ -80,6 +82,16 @@ export default function Home() {
 
     return (
         <IonContent>
+            {process.env.REACT_APP_LOCAL_DEVELOPMENT && (
+                <React.Fragment>
+                    <button
+                        onClick={() => signInWithAuthEmulator(authInstance)}>
+                        EMULATOR SIGN-IN
+                    </button>
+                    <Link to="premium-video">Go to Twilio Video</Link>
```

Comment:
missing translation

Reply:
Since the PR inserts lots of new labels, I'd prefer to address this one in a separate PR